ChatSecure/Classes/View Controllers/OTRComposeViewController.m

```diff
@@ -157,6 +157,15 @@ - (void)viewWillAppear:(BOOL)animated
                                                object:nil];
 }
 
+- (void)viewWillTransitionToSize:(CGSize)size withTransitionCoordinator:(id<UIViewControllerTransitionCoordinator>)coordinator
+{
+    // Resize UISearchBar manually - it doesn't do it on its own on device turn.
+    self.searchController.searchBar.frame = CGRectMake(0, 0, size.width, size.height);
+    [self.searchController.searchBar sizeToFit];
+
+    [super viewWillTransitionToSize:size withTransitionCoordinator:coordinator];
+}
+
```

Comment:
I'm guessing this could be solved by adding the proper constraints elsewhere, but this solution is probably less brittle given the wonkiness of UISearchController's searchBar when embedded in views other than tableHeaderViews.